### DISCUSSION TITLE
How to view actually videos

### DISCUSSION THREAD

**Discussion by Jmsharris** [community/community#35526] [Other Feature Feedback, Questions, & Ideas]
https://www.dropbox.com/s/mkr84k8j18xgzx9/memories_history.json?dl=0

**Comment by fadyat** [comment]
If I understand your question correctly, you need to parse this json and only take the object that has `"Media Type":"Video"`

**Comment by fadyat** [comment]
Something like this in Python:
```python
import json


def get_videos():
    media = json.load(open('memories_history.json')).get('Saved Media')
    return tuple(
        m for m in media
        if m.get('Media Type') == 'Video'
    )


if __name__ == '__main__':
    get_videos()

```

**Reply by aimore** [reply]
Try this
https://gist.github.com/napcae/bb74c2dab54b8e15babc3952e7ffbd07
